tests/src/test/java/org/wildfly/plugins/bootablejar/maven/goals/DevServerTestCase.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.plugins.bootablejar.maven.goals;
+
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * @author jdenise
+ */
+@Ignore("Can't run, we can't call start from the mvn tests, break tests.")
```

Comment:
Does this prevent `dev` mode flow tests from being executed or am I just misunderstanding?

Reply:
Yes this test does not seem to work. The issue seems to have to do with something overriding STDOUT. At some point we can look at a fix, but I have not done that as part of this PR.